src/core/search/block_list_test.cc

```diff
@@ -103,6 +160,131 @@ TYPED_TEST(BlockListTest, RandomNumbers) {
   }
 }
 
+class BlockListTest : public testing::Test {
+ protected:
+};
+
+TEST_F(BlockListTest, Split) {
```

Comment:
Just for the testing purposes global Split should be exposed via header file but I do not think it should be a class member.